src/views/OrderDetail.vue

```diff
@@ -206,12 +206,38 @@ export default defineComponent({
       facilities: [] as any,
       queryString: "",
       searchedProduct: {} as any,
+      canLeave: false,
       isParentProductUpdated: false
     }
   },
   ionViewDidEnter(){
    this.fetchFacilities();
   },
+  async beforeRouteLeave(to, from) {
+    const alert = await alertController.create({
+        header: this.$t("Leave page"),
```

Comment:
Add all the text to en.json

Reply:
The text is already added in en.json file